### PR TITLE
Issue 76: Prevent unnecessary DB queries

### PR DESCRIPTION
**Pull request by torkiljohnsen**

### CODE REVIEW

**Comment by vinkla** [comment]
Thanks for the pull request @torkiljohnsen! Please also push an update to the test suite to match your new changes.

**Comment by vinkla** [comment]
Thanks for the pull request @torkiljohnsen! Please push an update to the test suite to match your new changes.

File: src/Translatable.php [hunk 1]
```diff
@@ -105,6 +109,10 @@ protected function getTranslation(string $locale)
             return $this->translationCache[$locale];
         }
 
+        if (!$this->exists) {
+            return null;
```
Comment (torkiljohnsen) [inline src/Translatable.php]: StyleCI whines about this, but we need to return `null` at this point. Suggestions?

Comment (torkiljohnsen) [reply src/Translatable.php]: @vinkla

Comment (vinkla) [reply src/Translatable.php]: `return;` should be the equivalent of `return null;`

File: tests/TranslatorTest.php [hunk 2]
```diff
@@ -87,6 +87,13 @@ public function testSetLocale()
         $this->assertSame($article->translate()->title, 'Use the force Harry');
     }
 
+    public function testNonExistingReturnsNull()
+    {
+        $article = new Article(['name' => 'Use the force Harry']);
+
+        $this->assertNull($article->translate('no'));
```
Comment (vinkla) [inline tests/TranslatorTest.php]: Good work! I think we also need to redo some of the existing tests.

File: src/Translatable.php (hunk 1 above)
Comment (vinkla) [reply src/Translatable.php]: Okay, don't worry. I'll go over the syntax.

**Comment by vinkla** [comment]
Thanks @torkiljohnsen! I'll release this with the new Laravel 5.6 release coming in a few weeks.

**Comment by torkiljohnsen** [comment]
Great 👍

**Pull request merged**